pkg/cmd/controller/csr.go

```diff
@@ -0,0 +1,51 @@
+package controller
+
+import (
+	"context"
+	"github.com/openshift/cluster-policy-controller/pkg/operator"
+	"github.com/openshift/library-go/pkg/operator/csr"
+	"github.com/openshift/library-go/pkg/operator/events"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+)
+
+const (
+	controllerName = "csr-approver-controller"
+	// Hardcoded for the Monitoring Operator:
+	// https://github.com/openshift/cluster-monitoring-operator/blob/master/manifests/0000_50_cluster-monitoring-operator_03-role-binding.yaml#L25-L27
+	monitoringServiceAccountNamespace = "openshift-monitoring"
+	monitoringServiceAccountName      = "cluster-monitoring-operator"
+	monitoringCertificateSubject      = "/CN=prometheus"
```

Comment:
@s-urbaniak I would've expected `/CN=system:serviceaccount:openshift-monitoring:openshift-state-metrics`, no?

Reply:
sounds good to me. **does** it have to match the exact service account name? I may be missing something, I assumed this is more or less free-form.

Reply:
I don't think it has to match, although I think I would expect it to. Or at least prepend it with the `system:` prefix

Reply:
I think following the canonical k8s syntax as you suggested sounds good. It is very clear to what it maps to :+1:

Reply:
Done